tom/tom.py

```diff
@@ -29,8 +27,7 @@ class complimentBot(irc.IRCClient):
     owner_name = ""
     currentTime = 0
     default = 'burn berNs'
-    botList = [
-        "dad", "Seahorse", "pointbot", "botProtector",
+    botList = ["dad", "Seahorse", "pointbot", "botProtector",
```

Comment:
do you plan to use the bot_list.txt file?

Reply:
That's coming in the future.